packages/core-utils/src/provider.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Provider Utilities
+ */
+
+import { ethers } from 'ethers'
+
+// Create a fallback provider using a comma delimited
+// string of URLs
+export const FallbackProvider = (config: string) => {
+  const providers = []
+  const urls = config.split(',')
+  for (const url of urls) {
+    const provider = new ethers.providers.StaticJsonRpcProvider(url)
+    providers.push(provider)
+  }
+  return new ethers.providers.FallbackProvider(providers)
```

Comment:
looks good - note here that we can further parameterize things like weights of different providers by providing an array of  `FallbackProviderConfig` instead.
> FallbackProviderConfig. If a Provider is provided, the defaults are a priority of 1 and a weight of 1.

Reply:
https://github.com/ethers-io/ethers.js/blob/c2c0ce75039e7256b287f9a764188d08ed0b7296/packages/providers/src.ts/fallback-provider.ts#L97

The existing implementation will randomize which is used. We could set the priority based on the order of URLs passed in

Reply:
Does this do quorum by default?